resources/config/conf.yaml

```diff
@@ -0,0 +1,12 @@
+plugins:
+  trivy:
+    exit-code: 0
+    timeout: 5m0s
+    severity: HIGH,CRITICAL
```

Comment:
this looks like list but are not in list format for yaml. Maybe we should make this a yaml list?

Reply:
We are referring trivy buildkite plugin README.md